docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  pact_container_p:
+    build:
+      context: .
+      dockerfile: Dockerfile
```

nit: just need to specify the folder if only contains 1 docker file. Will use Dockerfile by defualt

 ```suggestion
    build: .
```